nucliadb_utils/nucliadb_utils/settings.py

```diff
@@ -125,6 +125,10 @@ class IndexingSettings(BaseSettings):
     index_jetstream_auth: Optional[str] = None
     index_local: bool = False
 
+    indexed_jetstream_target: str = "indexed.{partition}"
+    indexed_jetstream_group: str = "indexed-{partition}"
```

Comment:
Since we're not consumer from this component, maybe we shouldn't have this setting?